src/main/java/uk/gov/pay/adminusers/model/ForgottenPassword.java

```diff
@@ -18,28 +18,34 @@
     private Integer id;
     private String code;
     private ZonedDateTime date;
+    private String userExternalId;
```

Comment:
I doubt we ever need this attribute in `ForgottenPassword`. We should be able to do forgotten password without this thing